pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/user/UserDao.java

```diff
@@ -1,117 +1,63 @@
 package org.broadinstitute.dsm.db.dao.user;
 
-import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
-
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
 import java.util.Optional;
 
 import lombok.NonNull;
+import org.broadinstitute.ddp.db.TransactionWrapper;
 import org.broadinstitute.dsm.db.dao.Dao;
 import org.broadinstitute.dsm.db.dto.user.UserDto;
 import org.broadinstitute.lddp.db.SimpleResult;
 
 public class UserDao implements Dao<UserDto> {
 
-    public static final String USER_ID = "user_id";
-    public static final String NAME = "name";
-    public static final String EMAIL = "email";
-    public static final String PHONE_NUMBER = "phone_number";
-    private static final String SQL_INSERT_USER = "INSERT INTO access_user (name, email) VALUES (?,?)";
-    private static final String SQL_DELETE_USER_BY_ID = "DELETE FROM access_user WHERE user_id = ?";
-    private static final String SQL_SELECT_USER_BY_EMAIL =
-            "SELECT user.user_id, user.name, user.email, user.phone_number FROM access_user user WHERE user.email = ?";
-    private static final String SQL_SELECT_USER_BY_ID =
-            "SELECT user.user_id, user.name, user.email, user.phone_number FROM access_user user WHERE user.user_id = ?";
-
     public Optional<UserDto> getUserByEmail(@NonNull String email) {
-        SimpleResult results = inTransaction((conn) -> {
+        SimpleResult results = TransactionWrapper.withTxn(TransactionWrapper.DB.DSM, handle -> {
```

Comment:
should the transaction now point to dss?

Reply:
Will be changed in next branch ( it is in another ticket)